addons/web/static/src/model/relational_model/utils.js

```diff
@@ -219,6 +219,18 @@ export function extractFieldsFromArchInfo({ fieldNodes, widgetNodes }, fields) {
                         );
                         Object.assign(activeField.related.fields, formArchInfo.fields);
                     }
+
+                    if (fieldNode.views.default) {
+                        const defaultArchInfo = extractFieldsFromArchInfo(
```

Comment:
that's weird, because default isn't coming from an arch

Reply:
chek if it s already done if viewMode is "default"